vendor/vendor.json

```diff
@@ -21,10 +21,10 @@
 			"revisionTime": "2017-04-08T17:35:22Z"
 		},
 		{
-			"checksumSHA1": "i27YHcyHY4stkXm12/ztso3wHUo=",
+			"checksumSHA1": "h4FKEw19jw7+Je5OwsISRKIGTm8=",
 			"path": "github.com/codegangsta/cli",
```

Comment:
it seems like this tool had a name change, this now re-directs to `https://github.com/urfave/cli`, should we update it here?

Reply:
I was able to fetch it fine. Not sure how changing the name here would affect go imports